src/sfCommand.ts

```diff
@@ -333,10 +333,11 @@ export abstract class SfCommand<T> extends Command {
    *
    * @param message text to display.  Do not include a question mark.
    * @param ms milliseconds to wait for user input.  Defaults to 10s.
+   * @param defaultAnswer boolean to set the default answer to.  Defaults to true.
    * @return true if the user confirms, false if they do not.
    */
-  public async confirm(message: string, ms = 10000): Promise<boolean> {
-    return this.prompter.confirm(message, ms);
+  public async confirm(message: string, ms = 10000, defaultAnswer = true): Promise<boolean> {
```

Comment:
UIs usually ask for confirmation before doing an expensive/destructive operation and default to `no` to ensure the user really wants to do it.

Let's say my plugin has a bunch of commands that delete stuff in an org, I would have to set `defaultAnswer` to `no` on each command so it's easy to forget to do it.

Since the prompter is the one defaulting to `true` and it's public this change would require a major bump:
https://github.com/salesforcecli/sf-plugins-core/blob/main/src/ux/prompter.ts#L61

Reply:
@cristiand391 This is a utility function and it has no concept of what the implications are of a yes or no answer. Those decisions are best done by the caller. Making the default true is protecting backward compatibility.